Software/Python/easygopigo3.py

```diff
@@ -2519,7 +2519,9 @@ def __init__(self, port="I2C",gpg=None):
             raise
 
         try:
+            _grab_read()
             distance_sensor.DistanceSensor.__init__(self)
+            _release_read()
```

Comment:
I suppose we're still using the old way of I2C mutexing. 
We haven't got onto the newer one, I think. Am I right?

I think we postponed the newer version because there would be too many changes in one version.

Reply:
yes mutex changes were postponed for a bit